snomed/com.b2international.snowowl.snomed.api.rest/src/com/b2international/snowowl/snomed/api/rest/SnomedClassificationRestService.java

```diff
@@ -227,15 +232,24 @@ public SnomedClassificationRestService() {
 			@RequestParam(value="limit", defaultValue="50", required=false) 
 			final int limit) {
 		
+		final String expandWithRelationship;
+		if (Strings.isNullOrEmpty(expand)) {
+			expandWithRelationship = "relationship()";
+		} else if (expand.contains("relationsip")) {
```

Comment:
```suggestion
		} else if (expand.contains("relationship(")) {
```

The typo definitely needs fixing, the opening parenthesis ensures that this is the end of an expand option, it is just a bonus.